iosHyperskillApp/iosHyperskillApp/Sources/Modules/HomeSubmodules/ChallengeWidget/Views/ChallengeWidgetView.swift

```diff
@@ -0,0 +1,65 @@
+import shared
+import SwiftUI
+
+extension ChallengeWidgetView {
+    struct Appearance {
+        let backgroundColor = Color(ColorPalette.surface)
+    }
+}
+
+struct ChallengeWidgetView: View {
+    private(set) var appearance = Appearance()
+
+    let viewStateKs: ChallengeWidgetViewStateKs
+
+    let viewModel: ChallengeWidgetViewModel
+
+    var body: some View {
+        buildBody()
+    }
+
+    @ViewBuilder
+    private func buildBody() -> some View {
```

Comment:
nit: I don't see reasons to extract `buildBody` function here, `body` property doesn't contain any other code and we can substitute function call to it's body content

Reply:
>  I don't see reasons to extract buildBody function here

I can't find the resource where I read about this, but this is done in performance reason - it's always better to have a small `body` and extract some views into helper computed properties / functions or own files